compiler/ballerina-lang/src/test/java/io/ballerina/projects/ProjectUtilsTests.java

```diff
@@ -52,23 +54,35 @@ public void setUp() throws IOException {
     @Test
     public void testReadBuildJson() {
         Path buildFilePath = PROJECT_UTILS_RESOURCES.resolve(ProjectConstants.BUILD_FILE);
-        BuildJson buildJson = ProjectUtils.readBuildJson(buildFilePath);
-        Assert.assertEquals(buildJson.lastBuildTime(), 1629359520);
-        Assert.assertEquals(buildJson.lastUpdateTime(), 1629259520);
+        try {
+            BuildJson buildJson = ProjectUtils.readBuildJson(buildFilePath);
+            Assert.assertEquals(buildJson.lastBuildTime(), 1629359520);
+            Assert.assertEquals(buildJson.lastUpdateTime(), 1629259520);
+        } catch (Exception e) {
+            Assert.fail("Reading Build Json failed");
+        }
     }
 
     @Test(expectedExceptions = ProjectException.class,
-            expectedExceptionsMessageRegExp = "Failed to read the 'build' file")
+            expectedExceptionsMessageRegExp = "Failed to read the 'build' file", enabled = false)
```

Comment:
Why do we need to disable some tests?

Reply:
These tests actually call the `readBuildJson` function directly and assert for the exception message being thrown. With the change, the specific exception messages are not thrown so these tests fail. 

Should I remove them entirely or modify them in a certain way?

Reply:
We can assert for the original IO exceptions here, isnt it?

Reply:
Changed tests to assert for specific exceptions